compositorcontroller.cpp

```diff
@@ -105,8 +113,10 @@ namespace RdkShell
     double gNextInactiveEventTime = RdkShell::seconds() + gInactivityIntervalInSeconds;
     uint32_t gLastKeyCode = 0;
     uint32_t gLastKeyModifiers = 0;
+    uint64_t gLastKeyMetadata = 0;
     std::shared_ptr<RdkShellEventListener> gRdkShellEventListener;
     double gLastKeyPressStartTime = 0.0;
+    double gLastKeyRepeateTime = 0.0;
```

Comment:
Please remove the extra "e" in gLastKeyRepeateTime.  Please rename to gLastKeyRepeatTime to fix the typo.